packages/extension/src/components/DaSidebar.vue

```diff
@@ -5,15 +5,17 @@
       <div class="sidebar__filter">
         <da-mode-switch class="sidebar__filter_switch"
                         first-icon="link" second-icon="hashtag"
-                        :checked="filterChecked" @toggle="toggleFilter"/>
+                        :checked="filterChecked" @toggle="toggleFilter($event); $refs.searchTags.value = ''"/>
       </div>
+
       <div class="sidebar__element sidebar__tags__search btn btn-menu"
-           :class="{selected: searchFocused}" v-if="filterChecked">
+           :class="{selected: searchFocused}">
         <svgicon name="magnifying" class="sidebar__element__image"/>
-        <input class="sidebar__input" type="text" placeholder="Search tags" ref="searchTags"
-               @input="updateSearch"
+        <input class="sidebar__input" type="text" :placeholder="searchPlaceholder" ref="searchTags"
```

Comment:
I think it's better to change the name of the `ref`
```suggestion
        <input class="sidebar__input" type="text" :placeholder="searchPlaceholder" ref="search"
```